### ISSUE TITLE
o-header-services: improve markup for drop-down menus - ID: DAC_Headings_And_Labels_Name_Role_Value_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#551]
# Issue
This is from page 170 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-header-services@5.0.1&demo=drop-down-navigation&system_code=origami-repo-data&brand=master>

The ‘DOCS’ and ‘SPECS’ links in the navigation region have both the links and the expandable buttons wrapped inside the same list item. As a result, both the links and the buttons are keyboard focusable.
When users attempt to activate the links using a mouse, the expandable menus open and then quickly close again, and users are only able to open the menus by clicking on the adjacent buttons. The labels for the buttons have duplicated aria-labels which read ‘Toggle dropdown menu’ which screen reader users may find confusing as it is not clear what the dropdown menu is opening.
Additionally, when the buttons are activated and the menus expand, screen readers are not informed of this behaviour.

# Screen reader comments:

> “Exploring the Primary navigation region both in and out of context with the arrow keys and the ‘B’ keystroke to cycle between buttons, I found that activating toggle buttons did not report that each expands to reveal new content.
> This is likely to confuse some Screen Reader users because they will have no way of knowing that the button reveals expanded content aside from using the arrow keys.
> Including this information when the toggle buttons activate should make comprehension and navigation more efficient.”
> 
> “Navigating through the toggle buttons with the ‘B’ keystroke, I found that the toggle buttons were not labelled. All I heard when moving through these buttons was the phrase ‘toggle dropdown button’.
> This was confusing when viewed out of context and may also confuse other Screen Reader users as there is no way to determine to which menu the toggle button refers out of context.
> Ensuring that toggle buttons are clearly labelled should help to avoid confusion and make navigation easier.”

# Keyboard only user comments:
>  “When tabbing through navigation, I have to tab into the dropdown arrow itself to open it fully. This is unusual behaviour unusual would expect it to be encapsulated with the title for it to open as one. So that they know which dropdown correlates with which title.

# Proposed Solution:
It would benefit all users if the links / buttons were two, individual focusable elements as opposed to two links and two buttons. Further information can be found in [Fly-out Menus - W3.org](https://www.w3.org/WAI/tutorials/menus/flyout/).
